index.html

```diff
@@ -1738,88 +1738,65 @@
+exacerbated by other violations of privacy.
 
-Abusive online behavior may include: sending [=unwanted information=]; directing others to contact
-or bother a person ("dogpiling"); disclosing sensitive information about a person; posting false
-information about a person; impersonating a person; insults; threats; and hateful or demeaning
-speech.
+Harassment may include: sending [=unwanted information=]; directing others to contact
+or bother a person ("dogpiling"); disclosing [sensitive information](#sensitive-information) about a person; posting false information about a person; impersonating a person; insults; threats; and hateful or demeaning speech.
 
-Disclosure of identifying or contact information (including "doxxing") can be used, including by
-additional attackers, to send often persistent unwanted information that amounts to harassment.
-Disclosure of location information can be used, including by additional attackers, to intrude on a
+Disclosure of identifying or contact information (including "doxxing") can be used to send often persistent [=unwanted information=] that amounts to harassment.
+Disclosure of location information can be used to intrude on a
 person's physical safety or space.
 
-Mitigations for harassment include but extend beyond mitigations for unwanted information and other
-privacy principles. Harassment can include harmful activity with a wider distribution than just the
-target of harassment.
-
-<div class="practice">
-  <p>
-    <span class="practicelab" id="abuse-reporting">
-      Systems that allow for communicating on the Web must provide an effective capability to report
-      abuse.
-    </span>
-  </p>
-</div>
-
 Reporting mechanisms are mitigations, but may not prevent harassment, particularly in cases where
 hosts or intermediaries are supportive of or complicit in the abuse.
 
-<div class="note">
-  Effective reporting is likely to require:
-
-  * standardized mechanisms to identify abuse reporting contacts
-  * visible, usable ways provided by sites and user agents to report abuse
-  * identifiers to refer to senders and content
-  * the ability to provide context and explanation of harms
-  * people responsible for promptly responding to reports
-  * tools for pooling mitigation information (see Unwanted information, below)
-</div>
-
-## Unwanted Information {#unwanted-information}
-
-Receiving unsolicited information that either may cause distress or waste the recipient's
-time or resources is a violation of privacy.
+Effective reporting is likely to require:
 
-<div class="practice">
+* standardized mechanisms to identify abuse reporting contacts;
+* visible, usable ways provided by sites and user agents to report abuse;
+* identifiers to refer to senders and content;
+* the ability to provide context and explanation of harms;
+* people responsible for promptly responding to reports;
+* tools for pooling mitigation information (see [[[#example-10]]]).
```

Comment:
Use an explicit ID for links instead of the auto-generated one. If you take the id I suggested over there:

```suggestion
* tools for pooling mitigation information (see [[[#example-reducing-unwanted-information]]]).
```